libsolidity/interface/CompilerStack.h

```diff
@@ -289,7 +290,8 @@ class CompilerStack: boost::noncopyable
 	std::vector<Source const*> m_sourceOrder;
 	std::map<std::string const, Contract> m_contracts;
 	std::string m_formalTranslation;
-	ErrorList m_errors;
+	ErrorList m_errorList;
```

Comment:
I wouldn't say it is a convention. Small constructors are in the header as are small functions (unless they required some complicated non-forward-declared structure).